google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFrameworkDetector.java

```diff
@@ -44,6 +44,11 @@ public AppEngineFrameworkDetector() {
 
   @Override
   public void setupFacet(@NotNull AppEngineStandardFacet facet, ModifiableRootModel model) {
+    AppEngineWebIntegration.getInstance().setupRunConfiguration(
+        null /*artifact*/,
```

Comment:
If this is null here, how does the artifact get created? Is this something the user will have to configure manually?